keps/sig-scheduling/4816-dra-prioritized-list/README.md

```diff
@@ -323,7 +342,218 @@ required) or even code snippets. If there's any ambiguity about HOW your
 proposal will be implemented, this is the place to discuss them.
 -->
 
-A `DeviceRequest` that populates the `FirstAvailableOf` field must *not*
+For the `v1beta2` API, all fields in the current `DeviceRequest` field except
+`Name` will be moved into a new `SpecificDeviceRequest` type and the `DeviceRequest`
+type will get a field called `Exactly` using this type. So for a request that doesn't
+provide a prioritized list of alternatives, the user will provide the `Name` and
+then set the `Exactly` field which contains the details about the request.
+
+The `FirstAvailable` field is mutually exclusive with the `Exactly` field, and
```

Comment:
Other possible names:
* `FirstMatch`
* `FirstFit`
* `FirstOf`
* `OneOf`

I could imagine one day, once we have scoring, there being: `Exactly`, `FirstMatch`, `BestMatch`, etc.

Reply:
Yeah, so I did think about other names, but I haven't found anything that is clearly better than the naming we already have here. I think the names here are pretty good and communicates the intent pretty well.

I think I slightly prefer `FirstAvailable` over `FirstMatch`, `FirstFit` and `FirstOf` since it communicates clearly that it will pick the first one that is available. I don't think I like the `OneOf` option as it sounds more like a random entry will be chosen than the alternatives being tried in order.